test/search.test.js

```diff
@@ -72,12 +72,6 @@ test('applies modifier if possible', (t) => {
   t.ok(Object.keys(found.items).length > 0)
 })
 
-test('enables uid', (t) => {
```

Comment:
uids are not enabled anymore as it breaks custom ordering aka results scoring.

Reply:
That's going to be a deal breaker. We return the UID so that Alfred will prioritize results based on usage heuristics.

https://www.alfredapp.com/help/workflows/inputs/script-filter/json/
![image](https://user-images.githubusercontent.com/321201/151431745-14447a78-5aec-49ae-8667-8313666b4693.png)

Reply:
@jsumners Is this actually wanted for emoji search results? In my experience with `alfred-emoji` it leads to poor-quality results + inconsistent results ordering.

This makes it hard to input emojis fast because the ordering changes regularly for a given input. You can't just type your usual keyword and press Return. Instead, you have to make sure that the top result is actually the one you want, rather than another result that jumped on top because you used it a lot (possibly in another query).

Reply:
I think it is desired. I definitely rely upon it. Adding a toggle via an environment variable would be acceptable.

Reply:
@jsumners Good suggestion, I'll update the PR and ask for rereview once done.